GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
+   Subproject's Approvers. Each Subproject vote will be conducted by the
+   OF Election Officials and calculated by the Condorcet Method.
+3. Following the announcement of the Subproject results, the
+   Community Representatives are chosen by all Members of the Operator
+   Framework. The vote will be conducted by the OF Election Officials
+   and calculated by the Condorcet Method.
+
+If individuals are nominated to be Subproject Representative, the Subproject elections will be calculated in a random order. No individual can
+hold more than one seat on the Steering Committee.
+
+### Steering Committee Bootstrapping
+
+Upon ratification of this document, the 7 elected Chairs of the Operator
+Framework will become the Steering Committee for a period of one year,
+afterwards Steering Committee elections will proceed as outlined in this
+document.
+
+## Governance Amendments
+
+The Operator Framework Community may adopt or amend its Governing
+Documents according to the following process:
+
+1. Propose a change by opening an issue in the community, and if there
+   seems to be general agreement, open a pull request to this document.
+2. The change should be announced via the mailing list along with an invitation for discussion on the Pull Request.
+3. A minimum of 7 days must pass between the announcement and the vote,
+   and the vote must remain open for a minimum of 7 days. The Steering
+   Committee may extend both periods.
+
+For a vote to pass, all of the following must occur:
+* A quorum of 3 Steering Committee members is required.
+* A 2/3 majority of the Steering Committee quorum must approve.
+
+## Code of Conduct Committee
+
+In order to review and enforce the Code of Conduct, the Steering
+Committee selects 3 people to be on the Code of Conduct Committee.
+These individuals will be chosen based on their community management and
+code of conduct experience, with diverse representation across the
+committee, including employer, gender, race, background, and region of
+the world. To avoid fatigue, the Steering Committee will replace at
+least 2 members of the CoC Committee each year. Members of
+the committee do not need to be members of Operator Framework if they
+have sufficient other expertise.
+
+The CoC Committee will receive reports of conduct violations confidentially,
+and will discuss them in closed meetings. If a report is determined to be a
+violation, they will recommend action on it appropriate to the scale, nature,
+and context of the violation, from requiring an apology, up to expulsion of an
+individual from the project. In the event that a contributor is to be demoted
+or expelled, the CoC Committee will forward this recommendation to the Steering
+Committee who will ratify it in a closed meeting. Should a member of the
+Steering Committee be the offender or the reporter, this recommendation will
+instead be forwarded to the CNCF staff for final arbitration.
+
+## Adding New Subprojects
+
```

Comment:
when a new subproject is added (and they gain a steering committee seat) This seat will be additive, (no existing community chairs will be removed, even if there are fewer at large seats next time)